# possible values
int8 INACTIVE = 0
int8 SINGLE_PUSH = 1
int8 DOUBLE_PUSH = 2
int8 LONG_PUSH = 3


# fields
int8 play_pause
int8 stop
int8 custom
int8 up
int8 down